main.js

```diff
@@ -1,3 +1,26 @@
+// Global Variables
+var $viewList = ['#items', '#details', '#checkout']
+var $listItems = document.querySelector('#items')
+var $detailItems = document.querySelector('#details')
+var $checkoutItems = document.querySelector('#checkout')
+
+// Cart Amount
+var $cart = document.querySelector('#cart')
+var $cartTotal = document.createElement('span')
+$cartTotal.classList.add('badge', 'badge-info')
+$cart.appendChild($cartTotal)
+$cartTotal.textContent = shoppingCart.length
+
+// Change Views
+function changeView(viewList, activeView) {
+  viewList.forEach(function (views) {
```

Comment:
you're looping through each view, so you may want to rename this to `view` instead of `views